TUTORIAL.md

```diff
@@ -0,0 +1,33 @@
+# Guide to Solving Operators
+
+### `unsafe?`
+
+After running `rspec` for the first time we see we have seven tests failing.
+
```

Comment:
Feel free to also include "debugging" techniques. Like copy pasting the method into irb or using binding.pry to play with stuff. @SophieDeBenedetto do they know pry yet when operators come up?